test/scripts/deploy-and-run-regression.sh

```diff
@@ -18,7 +18,7 @@
 
 
 
- 
+
```

Comment:
we could even remove some lines here
```suggestion
```

Reply:
but maybe that is another pre-commit hook

Reply:
There are a large number hooks seen here: https://pre-commit.com/hooks.html

I can clean up more but I was trying to keep this pr focused on one type of issue.